smtlib-backends-process/tests/Examples.hs

```diff
@@ -79,10 +80,26 @@ manualExit = do
   -- gracefully
   --
   -- if this isn't enough for you, it is always possible to send an @(exit)@
-  -- command using 'Process.write', access the solver process using
-  -- 'Process.process' and kill it manually
-  -- if this is what you go with, don't forget to also cancel the
-  -- 'Process.errorReader' asynchronous process!
+  -- command using 'command_' (see the 'flushing' example), access the solver
+  -- process using 'Process.process' and kill it manually if this is what you go
+  -- with, don't forget to also cancel the 'Process.errorReader' asynchronous
+  -- process!
```

Comment:
For another PR, rather than discussing this in abstract, another example would be necessary (`moreManualExit`).

Reply:
Is it really necessary ? I'm mentioning it for completeness, but I don't see how anyone would ever need to do it like this.

Reply:
Either it is useful information, or it isn't. If it isn't useful, then there is no need to communicate it, and this aside can be removed. It has more potential to confuse than to help as it is.

Reply:
I guess the point was to indicate that it was still possible to send an `(exit)` command with `Process.write`, but now the new example covers this case. I'll remove the comment.